sftpretty/__init__.py

```diff
@@ -251,7 +251,7 @@ def _set_logging(self):
             console.setFormatter(console_formatter)
             getLogger().addHandler(console)
```

Comment:
I think this is the line that is causing duplicate logging down the line when using sftpretty. I think we want to use the 'SFTPretty' here instead of the root logger